docker/stable/ubuntu14.04/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:trusty
+MAINTAINER Andrew Schwartzmeyer <andschwa@microsoft.com>
+
+ARG POWERSHELL_RELEASE=v6.0.0-alpha.9
+ARG POWERSHELL_PACKAGE=powershell_6.0.0-alpha.9-1ubuntu1.14.04.1_amd64.deb
+
+# Setup the locale
+ENV LANG en_US.UTF-8
+ENV LC_ALL $LANG
+RUN locale-gen $LANG && update-locale
+
+# Install dependencies and clean up
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        libc6 \
+        libcurl3 \
+        ca-certificates \
+        libgcc1 \
+        libicu52 \
+        libssl1.0.0 \
+        libstdc++6 \
+        libtinfo5 \
+        libunwind8 \
+        libuuid1 \
+        zlib1g \
+        curl \
+        git \
```

Comment:
We keep curl and git as they are necessary for further bootstrapping, and are minimal in size.